src/Frame.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+function Frame() {
+  this.strike = false;
+  this.spare = false;
+  this.frameScore = {roll1: 0, roll2: 0};
+  this.skipSecondRoll = false;
```

Comment:
this.strike and this.skipSecondRoll are always gonna be the same, could suggest having only one of them or removing beth and just having a method which returns whether its a strike or not